README.md

```diff
@@ -1,38 +1,128 @@
 
-BentoMap is also compatible with [Carthage](https://github.com/Carthage/Carthage). To install it, simply add the following line to your Cartfile:
-
-```ogdl
+```
 github "Raizlabs/BentoMap"
 ```
 
-## Authors
+#### Manually
+1. Download all of the `.swift` files in `BentoMap/` and `BentoMap/Extensions/` and drop them into your project.  
+2. Congratulations!  
+
+## Usage example
+
+To see a full implementation of loading data into a map view, check out the [example project][ex].
+
+### Inserting Data
+
+```swift
+import BentoMap
+
+static var sampleData: QuadTree<Int, MKMapRect, MKMapPoint> {
+    var samples = QuadTree<Int, MKMapRect, MKMapPoint>(bentoBox: BentoBox(minPoint: MKMapPointForCoordinate(CLLocationCoordinate2D.minCoord), maxPoint: MKMapPointForCoordinate(CLLocationCoordinate2D.maxCoord)), bucketCapacity: 5)
+    let randomData = (1...5000).map { count in
+        return QuadTreeNode(originCoordinate: MKMapPointForCoordinate(CLLocationCoordinate2D.randomCoordinate()), content: count)
+    }
+    for node in randomData {
+        samples.insertNode(node)
+    }
+    return samples
+}
+
+```
+
+### Updating a Map View
+
+```swift
+func updateAnnotations(inMapView mapView: MKMapView,
+                                 forMapRect root: MKMapRect) {
+    guard !mapView.frame.isEmpty && !MKMapRectIsEmpty(root) else {
+        mapView.removeAnnotations(mapView.annotations)
+        return
+    }
+    let zoomScale = Double(mapView.frame.width) / root.size.width
+    let clusterResults = mapData.clusteredDataWithinMapRect(root,
+                                    zoomScale: zoomScale,
+                                    cellSize: Double(MapKitViewController.cellSize))
+    let newAnnotations = clusterResults.map(BaseAnnotation.makeAnnotation)
+
+    let oldAnnotations = mapView.annotations.flatMap({ $0 as? BaseAnnotation })
 
-Michael Skiba, mike.skiba@raizlabs.com
-Rob Visentin, rob.visentin@raizlabs.com
-Matt Buckley, matt.buckley@raizlabs.com
+    let toRemove = oldAnnotations.filter { annotation in
+        return !newAnnotations.contains { newAnnotation in
+            return newAnnotation == annotation
+        }
+    }
+
+    mapView.removeAnnotations(toRemove)
+
+    let toAdd = newAnnotations.filter { annotation in
+        return !oldAnnotations.contains { oldAnnotation in
+            return oldAnnotation == annotation
+        }
+    }
+
+    mapView.addAnnotations(toAdd)
+}
+```
+
+## Contributing
+
+Issues and pull requests are welcome! Please ensure that you have the latest [SwiftLint][sl] installed before committing and that there are no style warnings generated when building.
+
+Contributors are expected to abide by the [Contributor Covenant Code of Conduct][cc].
 
 ## License
 
-BentoMap is available under the MIT license. See the LICENSE file for more info.
+BentoMap is available under the MIT license. See the `LICENSE` file for more info.
+
+## Authors
+
+- Michael Skiba, mike.skiba@raizlabs.com
```

Comment:
#nth Twitter handles, if you want. I'll save you a Google search: Rob doesn't have one.